documentation/modules/proc-configuring-kafka-connect.adoc

```diff
@@ -43,3 +43,5 @@ spec:
 This can be done using `kubectl apply`:
 [source,shell,subs=+quotes]
 kubectl apply -f _your-file_
+
+. If simple authorization is enabled for Kafka Connect, xref:assembly-kafka-connect-authorization-{context}[configure the Kafka Connect user to enable access to the Kafka Connect consumer group and topics].
```

Comment:
Again ... if any authorization is enabled. Also, I think we should say _if authroization is enabled in Kafka brokers_ ... because you enable it per Kafka cluster. Not per Kafka Connect.

Reply:
> If authorization is enabled for Kafka Connect,